app/controllers/api/v1/messages_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class MessagesController < ApiController
+      def create
+        @message = conversation.messages.create!(message_params.merge({ user_id: current_user.id }))
+      end
+
+      private
+
+      def message_params
+        @message_params ||= params.permit(:body)
+      end
+
+      def conversation
+        @conversation = Conversation.find(params[:conversation_id])
```

Comment:
The current_user should be a participant of the conversation, and there should be tests for it as well

Reply:
Agree. We should probably return a `404` or `401` code if you're trying to get a conversation that doesn't belong to you (with a "Toxic alert" message 🤣 )